spec/suites/layer/tile/GridLayerSpec.js

```diff
@@ -1095,4 +1095,82 @@ describe('GridLayer', function () {
 		grid.redraw();
 		expect(wrapped.neverCalledWith(sinon.match.any, null)).to.be(true);
 	});
+
+	describe("Attribution adding / removing", function () {
+		var clock;
+		beforeEach(function () {
+			map.setView([0, 0], 1);
```

Comment:
The error in "Loads 8 kittens zoom 1" happens because of `map.setView([0, 0], 1);` ... I didn't find out why this happens. In my Browser over the karma `index.html` the tests are working, only on `PhantomJS 2.1.1 (Windows 8)` it breaks.
@mourner @johnd0e @IvanSanchez does someone of you know how to fix this in tests? I searched for 30 minutes

Reply:
The error:
> PhantomJS 2.1.1 (Windows 8) TileLayer number of kittens loaded Loads 8 kittens zoom 1 FAILED
        TypeError: null is not an object (evaluating 'this._map._fadeAnimated') (src/Leaflet.js:11510)